ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -3235,6 +3235,41 @@ struct TIndexBuildInfo: public TSimpleRefCount<TIndexBuildInfo> {
 
 };
 
+constexpr const char* Name(TIndexBuildInfo::EState state) noexcept {
```

Comment:
Consider moving this function to `ydb/core/tx/schemeshard/schemeshard_build_index__progress.cpp`, where is it really and only used.
Do you have practical usage for it some where else?

Reply:
Ok